documentation/adr/0008-use-pipeline-builds.md

```diff
@@ -34,11 +34,13 @@ The build jobs we be organised into `Horace` and `Herbert` folders. Separate pro
 ```
```

Comment:
Typo on the line above -which it won't let me comment on...
`The build jobs we be organised` -> `The build jobs will be organised`